atomics/T1112/T1112.yaml

```diff
@@ -1055,3 +1055,16 @@ atomic_tests:
     cleanup_command: |
        reg delete "HKCU\Software\Microsoft\Terminal Server Client" /v AuthenticationLevelOverride
     name: command_prompt
+
+- name: BlotchyQuasar RAT [aka Hive0129’s Banking Trojan] Defense Evasion Activities
```

Comment:
@PhyoPaingHtun : can we perhaps rename this atomic to something generic like : Disable UAC via registry keys?